### PR TITLE
Add new deployment via Ansible

### PR DESCRIPTION
**Pull request by Wenzel**
This PR reworks the way ccc repo is installed and deployed using Ansible playbook and collections

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
 .west/
+venv/
 Pipfile
 Pipfile.lock
 manifest/
 kafl/
 linux-guest/
 linux-host/
-tdvf/
+/tdvf/
```
Comment (il-steffen) [inline .gitignore:12]: typo?

Comment (Wenzel) [reply .gitignore:12]: This rule prevented me from adding the roles/tdvf/* files
by prefixing with a `/` this applies the rule only at the top-level of the repo

File: roles/guest/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    version: "{{ guest_revision | default(omit) }}"
+    force: yes
+
+- name: Copy BKC TDX guest kernel config
+  copy:
+    src: "{{ playbook_dir }}/bkc/kafl/linux_kernel_tdx_guest.config"
+    dest: "{{ guest_root }}/.config"
+    remote_src: yes
+
+- name: Enable Virtio harness
```
Comment (il-steffen) [inline roles/guest/tasks/main.yml]: I think this + kernel config is something to do via snakemake or similar pipeline. then call that pipeline for a test case.

Comment (Wenzel) [reply roles/guest/tasks/main.yml]: Yes, this is our configurable target, and it shouldn't be part of the Ansible setup.
I just wanted a quick PoC to validate fuzz.sh at this point.
Will have a look at snakemake / similar tools later on.

**Comment by Wenzel** [comment]
Reopen to test github actions until upstream merge

**Pull request merged**